Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift

```diff
@@ -414,10 +414,13 @@ extension TestListViewController: TestRunDelegate {
         displayTestRunAlert = true
     }
     
-    //TODO this method will be updated to take a status and unless the status is "in progress" the run will be removed.
-    func removeUpdatedItemFromTable() {
-        updatedRunName = testRunList.testRunList[selectedRunIndex].name
```

Comment:
We removed this assignment which handles what run name to show in the popup we present the user with, so we need this to be put back in.